### PR TITLE
Removed rails 2 from readme

### PR DESCRIPTION
**Pull request by maurogeorge**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,37 +12,13 @@ Put this in your Gemfile:
 gem 'paranoia'
 ```
 
-Then run `bundle`. Done.
```
Comment (radar) [inline README.md:15]: I think this line should stick around.

Comment (maurogeorge) [inline README.md:15]: I changed this line to a code block with `bundle install` as you can see on https://github.com/maurogeorge/paranoia/tree/readme-rails-2#installation--usage

Comment (radar) [inline README.md:15]: Oh, right you did.

**Comment by radar** [comment]
Thanks!

**Pull request merged**